packages/frontend/src/renderer/containers/widgets/channels/CreateChannel.tsx

```diff
@@ -15,10 +25,16 @@ export const CreateChannel = () => {
   const community = useSelector(communities.selectors.currentCommunityId)
   const channels = useSelector(publicChannels.selectors.publicChannels)
 
+  const communityErrors = useSelector(errors.selectors.currentCommunityErrors)
+  const error = communityErrors?.find(error => error.type === SocketActionTypes.CREATED_CHANNEL)
```

Comment:
Can't we have a selector for this (like "get by type")? This way one doesn't have to filter manually by error.type every time